DataCollector/CommandDataCollector.php

```diff
@@ -24,7 +23,7 @@ public function __construct(CommandLogger $commandLogger)
         $this->commandLogger = $commandLogger;
     }
 
-    public function collect(Request $request, Response $response, ?Throwable $exception = null)
+    public function collect(Request $request, Response $response, ?\Exception $exception = null)
```

Comment:
Needs a `phpcs:ignore` statement in the line above:
```
// phpcs:ignore SlevomatCodingStandard.Exceptions.ReferenceThrowableOnly
```

Also, `Exception` shouldn't be referenced by global name but rather imported above (as was done for `Throwable`). You can check whether your changes worked by running `composer install` and then `vendor/bin/phpcs`.